
{'date': 'Mon Mar 14 2022 18:39:19.909 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    Numer('jonh doe')
  module <module> line 16
    nums = [self.alka(ord(lt)- orda) for ws in words for lt in words]
TypeError: ord() expected a character, but string of length 4 found
'''},
{'date': 'Mon Mar 14 2022 18:40:08.666 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    Numer('jonh doe')
  module <module> line 18
    nums = [self.alka(ord(lt)- orda) for ws in words for lt in words]
TypeError: ord() expected a character, but string of length 4 found
'''},
{'date': 'Wed Mar 23 2022 12:13:18.477 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Rainbow()
TypeError: __init__() missing 1 positional argument: name
'''},
{'date': 'Wed Mar 23 2022 12:22:25.625 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  [Elemento(GREEN,x=ix*40 w=40, h=400-ix*20, cena=cena, style={'filter': f'hue-rotate({ix*20}deg)'} for ix in range(20))
                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 23 2022 12:22:57.243 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  [Elemento(GREEN,x=ix*40 w=40, h=400-ix*20, cena=cena, style={'filter': f'hue-rotate({ix*20}deg)'}) for ix in range(20)]
                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 23 2022 13:11:10.381 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  [tela <= svg.rect(x=ix*40, width="40", height=600-ix*30, style={"fill":'rgb(255,0,0)'},'filter': f'hue-rotate({ix*20}deg)'}) for ix in range(20)]
                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 23 2022 13:38:36.674 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 56
  [tela <= svg.rect(x=randint(0,ix)*40, x=randint(0,ix)*40, width="40", height=50,
                                         ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Wed Mar 23 2022 13:54:38.68 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 63
    Volpi()
  module <module> line 60
    [tela <= bandeira(x=randint(0,30)*40, y=randint(0,15)*50, width="40", height=50,
  module <module> line 52
    return path(d=f"M{x} {x} h40 v50 l-20 -20 l-20 20 v-40 Z", style=style)
NameError: name 'path' is not defined
'''},
{'date': 'Thu Mar 24 2022 11:37:30.99 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 86
    Volpi().main()
  module <module> line 65
    [self.tela <= self.bandeira(x=randint(0,30)*40, y=randint(0,15)*50, width="40", height=50,
NameError: name 'randint' is not defined
'''},
{'date': 'Thu Mar 24 2022 11:44:47.228 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 90
    Volpi().main()
  module <module> line 69
    [self.tela <= self.bandeira(x=randint(0,30)*40, y=randint(0,15)*50, width="40", height=50,
NameError: name 'randint' is not defined
'''},
{'date': 'Thu Mar 24 2022 11:46:47.967 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 90
    Volpi().main()
  module <module> line 69
    [self.tela <= self.bandeira(x=randint(0,30)*40, y=randint(0,15)*50, width="40", height=50,
  module <module> line 63
    return svg.path(d=f"M{x} {y} h40 v50 l-20 -20 l-20 20 v-40 Z", style=style)
NameError: name 'svg' is not defined
'''},